tests/test_multi.py

```diff
@@ -26,12 +26,14 @@
 from lsst.ctrl.oods.fileIngester import FileIngester
 import lsst.utils.tests
 import asynctest
+import utils
 
 
 class MultiComCamIngesterTestCase(asynctest.TestCase):
     """Test Scanning directory"""
```

Comment:
Since it showed up in the diff, does this docstring accurately describe the purpose of the test?

Reply:
Fixed